spec/requests/tenant_spec.rb

```diff
@@ -0,0 +1,96 @@
+  let(:catalog_admin) do
+    { :data => data, :meta => { :count => count } }
+  end
+
+  around do |example|
+    with_modified_env(:RBAC_URL => "http://localhost") do
+      ManageIQ::API::Common::Request.with_request(modified_request(org_admin)) { example.call }
+    end
+  end
+
+  describe 'POST /tenants/:id/seed' do
+    context "when the group was previously not seeded" do
+      let(:catalog_admin) do
+        { :data => [], :meta => { :count => 0 } }
+      end
+      before do
+        allow(api_instance).to receive(:list_group).with(group1.uuid).and_return(catalog_admin.to_json)
+        allow(RBAC::Seed).to receive(:new).and_return(rbac_seed)
+        stub_request(:get, "http://localhost/api/rbac/v1/groups/")
+          .to_return(:status  => 200,
+                     :body    => catalog_admin.to_json,
+                     :headers => response_headers)
+      end
+      before { post "#{api}/tenants/#{tenant_id}/seed", :headers => modified_headers(org_admin) }
+
+      it 'returns status code 200' do
+        expect(response).to have_http_status(200)
+        expect(JSON.parse(response.body)['message']).to eq "Tenant #{org_admin['identity']['account_number']} seeded"
+      end
+
+      it 'account number is populated in the RbacSeed table' do
+        expect(RbacSeed.find_by(:external_tenant => org_admin['identity']['account_number'])).to be_truthy
+      end
+    end
+
+    context "when the group already has been seeded" do
+      before do
+        RbacSeed.create!(:external_tenant => org_admin['identity']['account_number'])
+        allow(api_instance).to receive(:list_group).with(group1.uuid).and_return(catalog_admin.to_json)
+        stub_request(:get, "http://localhost/api/rbac/v1/groups/")
+          .to_return(:status  => 204,
+                     :body    => catalog_admin.to_json,
+                     :headers => response_headers)
+      end
+      before { post "#{api}/tenants/#{tenant_id}/seed", :headers => modified_headers(org_admin) }
+
+      it 'returns status code 204' do
+        expect(response).to have_http_status(204)
+        expect(response.body).to eq ""
+      end
+
+      it 'account number is in RbacSeed table' do
+        expect(RbacSeed.seeded(ManageIQ::API::Common::Request.current.user)).to be_truthy
+      end
+    end
+
+    context 'when the user is not an org admin' do
+      let(:catalog_admin) do
+        { :data => [], :meta => { :count => 0 } }
+      end
+      before do
+        allow(api_instance).to receive(:list_group).with(group1.uuid).and_return(catalog_admin.to_json)
+        allow(RBAC::Seed).to receive(:new).and_return(rbac_seed)
+        stub_request(:get, "http://localhost/api/rbac/v1/groups/")
+          .to_return(:status  => 200,
+                     :body    => catalog_admin.to_json,
+                     :headers => response_headers)
+      end
+      before { post "#{api}/tenants/#{tenant_id}/seed", :headers => default_headers }
+
+      it 'returns status code 200' do
```

Comment:
@syncrou is the status code 200 or 403 here? I am thinking a non org_admin user will not have access to seeding.